Analysis/HiggsTauTau/src/HTTRun2AnalysisTools.cc

```diff
@@ -540,14 +576,7 @@ namespace ic {
        "VVTo2L2Nu","ZZTo2L2Q","ZZTo4L",
        "WZTo2L2Q","WZTo1L3Nu","WZTo1L1Nu2Q"
       };
-    } else if(!is_fall15_){
```

Comment:
OK hard to tell from the diff - the selection inside is_fall15_ should be the default for 2015 now right? Not what was the default before which would have been for spring15